examples_tests/61.BlitFilterTest/alpha_test/alpha_test.glsl

```diff
@@ -0,0 +1,28 @@
+#ifndef _NBL_GLSL_BLIT_ALPHA_TEST_INCLUDED_
+#define _NBL_GLSL_BLIT_ALPHA_TEST_INCLUDED_
+
+#ifndef _NBL_GLSL_BLIT_ALPHA_TEST_MAIN_DEFINED_
+
+#include <../alpha_test/parameters.glsl>
+
+#ifndef _NBL_GLSL_BLIT_ALPHA_TEST_PASSED_COUNTER_DESCRIPTOR_DEFINED_
+#error _NBL_GLSL_BLIT_ALPHA_TEST_PASSED_COUNTER_DESCRIPTOR_DEFINED_ must be defined
+#endif
+
+nbl_glsl_blit_alpha_test_parameters_t nbl_glsl_blit_alpha_test_getParameters();
+
+float nbl_glsl_blit_alpha_test_getPaddedData(in ivec3 texCoords);
+
+void nbl_glsl_blit_alpha_test_main()
+{
+	const float alpha = nbl_glsl_blit_alpha_test_getPaddedData(ivec3(gl_GlobalInvocationID));
+
+	if (alpha > nbl_glsl_blit_alpha_test_getParameters().referenceAlpha)
```

Comment:
should be `>=` I think

Reply:
No.

[Took `CBlitImageFilter` as a reference](https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/asset/filters/CBlitImageFilter.h#L511).

Reply:
I'm saying it should be `>=` in both places probably.

Think what happens when you set reference alpha to 1